src/drivers/src/uart_syslink.c

```diff
@@ -56,7 +56,7 @@ static StaticSemaphore_t waitUntilSendDoneBuffer;
 static xSemaphoreHandle uartBusy;
 static StaticSemaphore_t uartBusyBuffer;
 static xQueueHandle syslinkPacketDelivery;
-STATIC_MEM_QUEUE_ALLOC(syslinkPacketDelivery, 8, sizeof(SyslinkPacket));
+STATIC_MEM_QUEUE_ALLOC(syslinkPacketDelivery, 16, sizeof(SyslinkPacket));
```

Comment:
Not sure if this will affect the rest of the STM at all?

Reply:
I will revert this since the increased queue size was to avoid asserts but this is handled by pausing interrupts instead which works well.